tests/utils.py

```diff
@@ -83,7 +84,9 @@ def all_concrete_subclasses(base: T_Type) -> list[T_Type]:
     return [
         sub
         for sub in all_subclasses(base)
-        if all(base is not abc.ABC for base in sub.__bases__) and not inspect.isabstract(sub)
+        if all(base is not abc.ABC for base in sub.__bases__)
+        and not inspect.isabstract(sub)
+        and all(parent not in {TypedNodeApply, TypedEdgeApply, TypedEdge, TypedNode} for parent in sub.__mro__)
```

Comment:
Why shouldn't these be tested?

Reply:
They should, they actually showed a bug. A was thinking these should be treated differently.

Reply:
Added a comment on it now, but it is that the `FakeCogniteResourceGenerator` does not support the descriptors.